lib/actions/ui.js

```diff
@@ -336,13 +341,18 @@ export function setLocale(locale) {
   return async function (dispatch, getState) {
     const { config } = getState().otp
     const { loggedInUser } = getState().user
-
     const { language: customMessages } = config
+    const configLocales = getConfigLocales(customMessages)
     const effectiveLocale = locale || getDefaultLocale(config, loggedInUser)
-    const messages = await loadLocaleData(effectiveLocale, customMessages)
+    const matchedLocale = getMatchingLocaleString(effectiveLocale, 'en-US')
```

Comment:
Not sure what happened here, the locale/language switching does not seem to work.
Also, using the URL to switch the locale (e.g. by appending `?locale=fr`) no longer works.